src/app/shared/components/users-billing-info/users-billing-info.component.html

```diff
@@ -125,10 +126,17 @@ <h3 class="form-step-card-title text-dark mb-3">
                         <strong>{{upgradeAmount|currency:currency}}</strong>
                       </div>
                     </div>
-
+                  </div>
                 </div>
               </div>
-              <div class="form-step-card" *ngIf="paymentMethod===paymentMethodType.BITCOIN && planType!==planTypeEnum.FREE">
+              <div class="monero"*ngIf="Monero">
```

Comment:
- place the space between `class="monero"` and `*ngIf="Monero"`
- it seems `Monero` in `*ngIf="Monero"` is variable, should be renamed with `monero` or something like that...
  follow the `camelCase` for the variable name, don't use `PascalCase`, it means the first character should be replaced with lower case.
  I think you should learn about `camelCase` and `PascalCase`